### PR TITLE
Fix adding torrent function

### PR DESCRIPTION
**Pull request by Micdu70**
Is there a valid reason to comment `$client->referer = $rt->referer;` in `plugins/loginmgr/accounts.php` (line 24)?
Because it's the easiest way to fix this => https://github.com/Novik/ruTorrent/pull/2092#issuecomment-675395900

### CODE REVIEW

**Comment by Novik** [comment]
Yes. Some search engines require concrete referer for processing. For example, for ZamundaNet this referer must include parameters of current search. But if we load referer from cache then authorization request will not contain this referer.

See this commit for details
https://github.com/Novik/ruTorrent/commit/cfad7fcd8f977b021f9a6fc000425bed67b2a064

**Comment by Micdu70** [comment]
Done! Thanks for your help @Novik 👍

File: plugins/loginmgr/accounts.php [hunk 1]
```diff
@@ -112,9 +117,9 @@ public function check( $client, $login, $password, $auto )
 		$modified = privateData::getModified($this->getName());
 		if( ($modified===false) || ((time()-$modified)>=$auto))
 		{
-			$data = privateData::load( $this->getName() );
-			if($this->login($client,$login,$password,$url,$method,$content_type,$body,$is_result_fetched) && 
-				$client->status>=200 && 
+			$data = $this->loadData($client);
```
Comment (Novik) [inline plugins/loginmgr/accounts.php:120]: IMHO must be

```suggestion
			$data = $this->loadData();
```

Comment (Micdu70) [reply plugins/loginmgr/accounts.php]: OK ;)

**Pull request merged**